src/Types/ProcessingFigures.tsx

```diff
@@ -64,7 +64,9 @@ export class TriangleFigure extends AnimatedFigure {
   }
 
   display() {
-    this.p5.noStroke();
+    // this.p5.noStroke();
+    this.p5.noFill();
+    this.p5.stroke(100, 200, 240);

```

Comment:
When you select a triangle it removes the fill, so if you then try to place either a rectangle or circle they are not visible.